utils/reactiveApi.ts

```diff
@@ -8,5 +8,18 @@ export default function reactiveApi<Data, Error>(data: AsyncData<Data, Error>) {
     },
   );
 
+  watch(data.error, async () => {
```

Comment:
I'd do this differently; specially because this has nothing to do with being reactive, and therefore, some of the API calls are not wrapped by this function.

I'd either create a separate wrapper or see whether we can integrate this with some way of error-handling the API-package already has in store.

Reply:
once the bmm-sdk correctly calls onError we can remove it.

Reply:
My point here also is a different one. By this, you're not catching the errors of all API calls, just those wrapped into this function. Some are not wrapped into this function.

Reply:
I'm aware of that. That's why it's only temporary until onError works. But 80% of errors is better than 0%.